src/pages/index.tsx

```diff
@@ -1,14 +1,12 @@
 import { type NextPage } from "next";
-import React, { ReactNode, useState } from "react";
 import { signIn, signOut, useSession } from "next-auth/react";
 import Head from "next/head";
-import Link from "next/link";
+import React, { ReactNode, useState } from "react";
+import Select, { StylesConfig, createFilter, defaultTheme } from "react-select";
 import Button from "~/components/Button";
 import Input from "~/components/Input";
-import { api } from "~/utils/api";
 import TopMenu from "~/components/TopMenu";
-import Select, { createFilter, StylesConfig } from "react-select";
-import { defaultTheme } from "react-select";
+import { api } from "~/utils/api";
```

Comment:
We should look into import-sorting being part of formatting.

Reply:
I think it'll make diffs less cluttered.

Reply:
I believe Wayne wanted a different sort for the import, that isn't alphabetical lol. Yesterday Wayne and I decided to go with a different dropdown component, so this is all going to change.

Reply:
The different sort should only affect Tailwind. It'd be nice to have auto cleanup of imports.

Reply:
Okay, I'll close this then, since it seems like we're abandoning this component.